lua/copilot/panel.lua

```diff
@@ -0,0 +1,500 @@
+local api = require("copilot.api")
+local util = require("copilot.util")
+
+local mod = {}
+
+local marker_prefix = "<!-- copilot "
+local marker_suffix = " -->"
+
+local panel_uri_prefix = "copilot://"
+
```

Comment:
Add this so that the markdown code block delimiters don't show

```suggestion

vim.treesitter.query.parse_query("markdown", [[
  (fenced_code_block_delimiter) @conceal (#set! conceal "")
]])

```

Reply:
I'm not familiar with the `vim.treesitter.query.parse_query` api. Does it have affect globally? We might not want to change any global settings.

On related note, I only used markdown because with it's possible to inject any language inside it. If there's any better alternative I'd gladly switch to it. You mentioned something like:

> I think it would be better to add an injection so treesitter highlights suggestion delimiter as html/markdown or something, and rest as &filetype, so that the number of problems it can run into are limited.

Can you explain that a bit?

Reply:
Nevermind, pushed an alternative approach. Markdown is not needed anymore.